### ISSUE TITLE
Training new landmark detection model

### ISSUE THREAD

**Issue by caiqi** [TadasBaltrusaitis/OpenFace#72]
Is it possible to train new landmark detection model with different landmark number.  For example, I have two dataset one with 21 landmarks and another with 68 landmarks. Can I use 21 landmark dataset to improve the performance of 68 landmark detection. Thanks!

**Comment by TadasBaltrusaitis** [comment]
Yes, it is possible you will just need to follow the instructions in 'matlab_version/readme.txt', it will require some time and a bit of change of code, but it should be reasonably documented.
